src/config/oauth.json

```diff
@@ -4,6 +4,10 @@
   "api_server_base_path": "{{ .Env.K8S_API_SERVER_BASE_PATH }}",
   "ws_api_server": "{{ .Env.WS_K8S_API_SERVER }}",
   "openshift_console_url": "{{ .Env.OPENSHIFT_CONSOLE_URL }}",
+  "forge_api_url": "{{ .Env.FABRIC8_FORGE_API_URL }}",
```

Comment:
It does, but I tried to keep some separation between the codebases, so that as we update the vars needed by the runtime console, we can easily just copy the updated oauth.json over to this codebase and not worry about patching stuff. TBH we should really just be reusing oauth.json from fabric8-runtime-console rather than hold a copy, but not sure how to do that atm, requires some build magic.